src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
```

Comment:
Ensure that the `comment` prop is being passed correctly to the `CommentInfo` component. This prop should contain `name`, `email`, and `body` fields.